src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -322,13 +325,14 @@ public function getCatalogProductList($offset, $limit, $orderBy, $sortOrder, $po
             'sql_select' => &$sqlSelect,
             'sql_table' => &$sqlTable,
             'sql_where' => &$sqlWhere,
+            'sql_group_by' => &$sqlGroupBy,            
             'sql_order' => &$sqlOrder,
             'sql_limit' => &$sqlLimit,
         ));
 
-        $sql = $this->compileSqlQuery($sqlSelect, $sqlTable, $sqlWhere, $sqlOrder, $sqlLimit);
-        $products = Db::getInstance()->executeS($sql, true, false);
-        $total = Db::getInstance()->executeS('SELECT FOUND_ROWS();', true, false);
+        $sql = $this->compileSqlQuery($sqlSelect, $sqlTable, $sqlWhere, $sqlGroupBy, $sqlOrder, $sqlLimit);
+        $products = \Db::getInstance()->executeS($sql, true, false);
+        $total = \Db::getInstance()->executeS('SELECT FOUND_ROWS();', true, false);
```

Comment:
No need to replace `Db` with `\Db`, as we declared the class at the top of the file with 

```php
use Db;
```